setup.cfg

```diff
@@ -46,14 +46,14 @@ install_requires =
     ogr
     packaging
     python-gnupg
-    rebasehelper
     requests
     requests-kerberos
     setuptools
     specfile
     tabulate
     bodhi-client
     koji
+    rpkg
```

Comment:
one cannot install rpkg from PyPI easily:
```
  Downloading pycurl-7.19.0.1.tar.gz (86 kB)            
     |████████████████████████████████| 86 kB 27.1 MB/s 
    ERROR: Command errored out with exit status 1:      
     command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-i_z9dj_3/pycurl_5092a9e20bf64242b1640b338e9b6436/setup.py'"'"'; __file__='"'"'/tmp/pip-install-i_z9dj_3/pycurl_5092a9e20bf6
4242b1640b338e9b6436/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"')
;f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-holvg1wi
         cwd: /tmp/pip-install-i_z9dj_3/pycurl_5092a9e20bf64242b1640b338e9b6436/
    Complete output (6 lines):                          
    /bin/sh: line 1: curl-config: command not found 
    Traceback (most recent call last):                  
      File "<string>", line 1, in <module>      
      File "/tmp/pip-install-i_z9dj_3/pycurl_5092a9e20bf64242b1640b338e9b6436/setup.py", line 90, in <module>
        raise Exception("`%s' not found -- please install the libcurl development files or specify --curl-config=/path/to/curl-config" % CURL_CONFIG)                                                                                           Exception: `curl-config' not found -- please install the libcurl development files or specify --curl-config=/path/to/curl-config
```

but since Packit requires fedpkg, that should be okay

Reply:
That's right, `curl-devel` (or a similar package) has to be installed. This was actually the reason we dropped `pycurl` a long time ago in rebase-helper (in favor of `requests`), but over time it brought a lot of issues with different lookaside cache servers and we had to implement a lot of nasty hacks to overcome them. I think in the end using `rpkg` (and thus `pycurl`) is a better alternative.